common/src/main/java/ch/admin/bag/covidcertificate/common/qr/QrScanFragment.kt

```diff
@@ -184,6 +181,14 @@ abstract class QrScanFragment : Fragment() {
 				e.printStackTrace()
 			}
 		}, mainExecutor)
+		
+		val metricPointFactory = qrCodeScanner.meteringPointFactory
+		val centerX = cutOut.left + cutOut.width / 2.0f
+		val centerY = cutOut.top + cutOut.height / 2.0f
+		val point = metricPointFactory.createPoint(centerX, centerY)
+		val action = FocusMeteringAction.Builder(point).build()
+		camera?.cameraControl?.startFocusAndMetering(action)
```

Comment:
Shouldn't this be done inside the cameraProviderFuture to ensure this is actually called and not ignored because the camera isn't initialized yet?